dialects/postgresql/src/main/kotlin/app/cash/sqldelight/dialects/postgresql/PostgreSqlTypeResolver.kt

```diff
@@ -141,6 +155,13 @@ class PostgreSqlTypeResolver(private val parentResolver: TypeResolver) : TypeRes
       literalValue.text.startsWith("INTERVAL") -> IntermediateType(PostgreSqlType.INTERVAL)
       else -> parentResolver.resolvedType(this)
     }
+    is PostgreSqlExtensionExpr -> when {
+      windowFunctionExpr != null -> {
+        val windowFunctionExpr = windowFunctionExpr as WindowFunctionMixin
```

Comment:
I'm surprised this works with the typo in `PostgreSql.bnf`

Reply:
or this code is never executed because we don't try to generate types based on the window function in tests anywhere

Reply:
or this code is never executed because we don't try to generate types based on the window function in tests anywhere

Reply:
Well, I guess, this explains the failures... I will it.